coveo-labs/autoreply-coveo-slack-bot
# feat(lambda): automate deployment of lambda from local dev setup

### Proposed Changes

Add release script for lambda deployment. Steps will be added later on in the readme

### How to test

1. Make sure you have the proper rights to deploy lambda, s3 bucket, API gateway 
2. `run npm run init && npm run deploy:lambda` and make sure it's deployed to AWS

### Scope

npm run script & lambda deployment

### Acceptance Criteria

- [x] The scope of the changes is clearly identified
- [ ] [README.md](https://github.com/coveo-labs/autoreply-coveo-slack-bot/blob/main/README.md) is adjusted to reflect the proposed changes (if relevant)
